widget/menu_item.go

```diff
@@ -245,6 +271,24 @@ func (r *menuItemRenderer) Refresh() {
 		r.background.Hide()
 	}
 	r.background.Refresh()
+
+	if r.i.Item.Disabled {
+		r.text.Color = theme.DisabledColor()
+		// TODO: enable after fixing translucence and alpha channel (https://github.com/fyne-io/fyne/issues/1977)
+		// r.checkIcon.Translucency = 0.7
+	} else {
+		r.text.Color = theme.ForegroundColor()
+		r.checkIcon.Translucency = 0
```

Comment:
Shouldn't be needed - see above, it will be toggling disabled type for the resource instead.